ai/STA/Tactic/place_ball.py

```diff
@@ -44,7 +44,9 @@ def __init__(self, game_state: GameState, player: Player,
         self.steady_orientation = None
 
     def _fetch_ball(self):
-        if self.game_state.field.is_outside_wall_limit(self.game_state.ball_position) or self._check_success():
+        #if self.game_state.field.is_outside_wall_limit(self.game_state.ball_position) or \
+        #        self._check_success():
```

Comment:
This code is only useful when doing ball placement at ulaval. Should we add a parameter to the field configuration to change the limit of the field?

Reply:
We could also just make the border width provided by ssl-vision smaller

Reply:
ouin, pense que c plus dans ssl vision qu'on devrait dealer ak ca, l'ia se fit aux data de geometry de la vision pour ca, si on commence a tapponer ak des offset un peu partout on va se faire rekt.